### PR TITLE
x11vnc: new article

### PR DESCRIPTION
**Pull request by mcpcpc**
(hopefully all wiki links are formatted correctly this time)

### CODE REVIEW

File: software/x11vnc.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+________________________________________________________________________________
+
+Refer to the @/xorg/xorg-server page for X server and Window Manager setup and 
+configuration.
+
+
+Configuration
+________________________________________________________________________________
+
+First verifying that you have x11vnc installed:
```
Comment (dylanaraps) [inline software/x11vnc.txt:19]: `veriifying` -> `verify`

**Comment by mcpcpc** [comment]
Just had an “oh duh” moment. This should probably be under xorg/, not software/ ?

**Comment by dylanaraps** [comment]
:+1:

**Comment by mcpcpc** [comment]
> 👍

moved!

**Comment by dylanaraps** [comment]
Thanks :+1:

**Pull request merged**